### DISCUSSION TITLE
Install optional components later

### DISCUSSION THREAD

**Discussion by jamesdeluk** [Security-Onion-Solutions/securityonion#1943] [Unsupported Versions]
Whenever I try to install SO with all components, the install fails. However, when I install with none, it succeeds.

Is it possible to add these components after install?

![image](https://user-images.githubusercontent.com/27723508/99348444-51354700-28e5-11eb-92d1-8f6ab586d584.png)

Thanks.

**Comment by weslambert** [comment]
Some components rely on information provided during setup to be configured correctly, although for others, they can be enabled via pillar files modification after setup.

**Reply by jamesdeluk** [reply]
Thanks @weslambert. Is there any documentation on which ones can be enabled afterwards, and if so how?
